catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -488,17 +491,19 @@ private void cloneSpriteVariables(Scene currentScene, Sprite cloneSprite) {
 		}
 	}
 
+	private void cloneLook(Sprite cloneSprite) {
+		cloneSprite.look = this.look.copyLookForSprite(cloneSprite);
+		if (cloneSprite.getLookDataList().size() > 0) {
+			cloneSprite.look.setLookData(cloneSprite.getLookDataList().get(0));
```

Comment:
This sets the look always to the first one of the `lookDataList`, regardless of the original clone's look.
would it make sense to replace it with something like
``` java
cloneSprite.look.setLookData(cloneSprite.lookList.get(this.lookList.indexOf(this.look.getLookData())));
```
(not tested...)